src/sentry/api/endpoints/organization_member_unreleased_commits.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import
+
+from django.db import connections
+from itertools import izip
+
+from sentry.api.bases import OrganizationMemberEndpoint
+from sentry.api.serializers import serialize
+from sentry.models import Commit, Repository, UserEmail
+
+query = """
+select c1.*
+from sentry_commit c1
+join (
+    select max(c2.date_added) as date_added, c2.repository_id
+    from sentry_commit as c2
+    join (
+        select distinct commit_id from sentry_releasecommit
+        where organization_id = %%s
+    ) as rc2
+    on c2.id = rc2.commit_id
+    group by c2.repository_id
+) as cmax
+on c1.repository_id = cmax.repository_id
+where c1.date_added > cmax.date_added
+and c1.author_id IN (
+    select id
+    from sentry_commitauthor
+    where organization_id = %%s
+    and lower(email) IN (%s)
```

Comment:
It might be worth noting that Django's default behavior of `iexists` and friends is to use `UPPER()` instead of `lower()` so if we wanted to add a functional index to this column, which I think we should, we'd likely do it over `upper(email)` so that Django's stuff works correctly.